pyanaconda/modules/payload/shared/utils.py

```diff
@@ -0,0 +1,43 @@
+#
+# Utility functions shared for the whole payload module.
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from pyanaconda.flags import flags
+from pyanaconda.core.util import getSysroot, mkdirChain
+
+
+def create_root_dir():
+    """Create root directory on the installed system."""
+    mkdirChain(getSysroot() + "/root")
+
+
+def write_module_blacklist():
```

Comment:
Same here.

Reply:
Does we really need to complicate all the calls just because of testing which can be easily patched?

Reply:
After our discussion we will take a parameter to make it consistent with the rest of the code.

Reply:
Fixed.